services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Using `get()` without handling exceptions can lead to `DoesNotExist` errors if the `MovieSession` is not found. Consider using `get_object_or_404()` or handling the exception to provide a more user-friendly error message.